### PR TITLE
feat: add header-checker-lint options file

### PR DESCRIPTION
**Pull request by chingor13**
The bot now reads a `.bots/header-checker-lint.json` file for configuration. If available, it will set the override the default configuration with values from this json file. If not available, it will use the default configuration.

### CODE REVIEW

File: packages/header-checker-lint/README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
 
-## Setup
+## Installation
+
+You can install this bot [from GitHub][github-app-link].
+
+### Configuration
+
+To configure the bot, you can create a configuration file:
+`.bots/header-checker-lint.json`. The contents of this file define a JSON object
```
Comment (bcoe) [inline packages/header-checker-lint/README.md]: clever, I like the `.bots/` folder, and then each config just maps to a bot?

File: packages/header-checker-lint/src/header-checker-lint.ts [hunk 2]
```diff
@@ -39,15 +40,72 @@ interface LicenseHeader {
...
+  }
+
+  static async fromGitHub(
+    path: string,
+    owner: string,
+    repo: string,
+    ref: string,
+    github: GitHubAPI
+  ): Promise<Configuration> {
+    return github.repos
```
Comment (bcoe) [inline packages/header-checker-lint/src/header-checker-lint.ts]: no need for the promise chain here, in an async function, you can just:

```
try {
await response = github.repos.getContents();
} catch () {}
```

Comment (chingor13) [reply packages/header-checker-lint/src/header-checker-lint.ts]: done

File: packages/header-checker-lint/README.md (hunk 1 above)
Comment (chingor13) [reply packages/header-checker-lint/README.md]: Yeah, this layout is totally up for discussion, though.

**Pull request merged**